paddle/fluid/platform/device_context_test.cu

```diff
@@ -43,9 +43,6 @@ TEST(Device, CUDADeviceContext) {
     ASSERT_NE(nullptr, gpu_device);
     cudnnHandle_t cudnn_handle = device_context->cudnn_handle();
     ASSERT_NE(nullptr, cudnn_handle);
-    cublasHandle_t cublas_handle = device_context->cublas_handle();
```

Comment:
cublasHandle_t would be tested in all cublas unittests. I just remove `cublas_handle()` to prevent users from calling cublas functions without lock guard.